src/benchmarks/micro/runtime/Math/Functions/Double/MaxDouble.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using BenchmarkDotNet.Attributes;
+
+namespace System.MathBenchmarks
+{
+    public partial class Double
+    {
+        // Tests Math.Min(double) over 5000 iterations for the domain -1, +1
+
+        private const double maxDelta = 0.0004;
+        private const double maxExpectedResult = -1.0;
+
+        [Benchmark]
+        public void Max() => MaxTest();
+
+        public static void MaxTest()
+        {
+            var result = 0.0; var val1 = -1.0; var val2 = -1.0 - maxDelta;
```

Comment:
I know that the existing benchmarks use `var` a lot, but I think that it would be better to use explicit type.

```suggestion
            double result = 0.0, val1 = -1.0, val2 = -1.0 - maxDelta;
```